.clang-format

```diff
@@ -1,18 +1,6 @@
 ---
 # See: https://clang.llvm.org/docs/ClangFormatStyleOptions.html
 BasedOnStyle: LLVM
-AlignAfterOpenBracket: DontAlign
-AlignEscapedNewlines: DontAlign
-AllowShortCaseLabelsOnASingleLine: true
-AllowShortIfStatementsOnASingleLine: true
-AlignConsecutiveAssignments: false
-AlignConsecutiveDeclarations: false
-AlignOperands: false
-AlignTrailingComments: false
```

Comment:
Comment alignment needlessly turns small code changes into large code changes.

Reply:
> Comment alignment needlessly turns small code changes into large code changes.

Not just comments. clang-format by default loves to align things, function call arguments, declarations, backslashes at the end of multi-line macros.

The main difference we see between clang-format 7 and 8 is how lambdas in `common::visitors` are now sometimes aligned. I couldn't find a way to turn that one off.

Reply:
With `BasedOnStyle=LLVM`, `AlignConsecutiveAssignments` and `AlignConsecutiveDeclarations` are already false.

Regarding `AlignTrailingComments`, I don't recommend using trailing comments at all.

For the others, I don't think it's a big issue. In cases such as
```
  int ASomewhatLengthVariableName = aaaaaaaa +
                                                                 bbbbbbb;
```
changing to 
```
  int ShorterName = aaaaaaaa +
                                                                 bbbbbbb;
```
where the `bbbbbbb` would change as well. If the merge-tool ignores whitespace, it would no even show up.